rcljava/src/main/java/org/ros2/rcljava/parameters/ParameterVariant.java

```diff
@@ -161,7 +161,7 @@ public final boolean asBool() {
     return this.value.getBoolValue();
   }
 
-  public final List<Byte> asByteArray() {
+  public final byte[] asByteArray() {
```

Comment:
For convenience, we could also add a new method, `asByteList()`, but users will incur a performance hit from boxing the primitive `byte` to `Byte`. Originally, I thought it would be better to discourage users from doing this, by having them do it themselves if they really want a List.